packages/liveblocks-react/src/factory.tsx

```diff
@@ -779,6 +789,38 @@ Please see https://bit.ly/3Niy5aP for details.`
     return useHistory().redo;
   }
 
+  function useCanUndo(): boolean {
+    const room = useRoom();
+    const [canUndo, setCanUndo] = React.useState(false);
```

Comment:
```suggestion
    const [canUndo, setCanUndo] = React.useState(room.history.canUndo);
```

Reply:
To be clear, this should be read as `React.useState(() => room.history.canUndo())`, i.e. using a computed initial value, by invoking that `canUndo` function on history.